Nos/Views/NIP05View.swift

```diff
@@ -36,23 +36,20 @@ struct NIP05View: View {
             }
             .task(priority: .userInitiated) {
                 if let nip05Identifier = author.nip05, let publicKey = author.publicKey {
-
-                    let isVerified: Bool
+                    let isVerified: Bool?
                     do {
                         isVerified = try await namesAPI.verify(
                             username: nip05Identifier,
                             publicKey: publicKey
                         )
-                    } catch URLError.cancelled {
```

Comment:
I'm reverting the logic here. Instead of showing the strikethrough in all errors and catch those in which we don't want to display the strikethrough, I'm now catching those we do want to display the strikethrough. The reason is because the list of URLError codes is very large (https://developer.apple.com/documentation/foundation/urlerror) and in most of them we don't want to display the strikethrough, also, false negatives are worse for the user experience than false negatives (its very bad to display a strikethrough for a user that has a valid nip-05).

So, from now on, if we detect another case for which we should display the strikethrough, we should add those in this try and catch. A added two that could be thrown if the user as a NIP-05 for a server that doesn't exist or for a server that is not accepting connections.